server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -438,12 +438,24 @@ export class ScribeLambda implements IPartitionLambda {
         }
     }
 
+    /**
+     * Updates protocol head to the new summary sequence number
+     * Protocol head is the sequence number of the last summary
+     * @param protocolHead The sequence number of the new summary
+     */
     private async updateProtocolHead(protocolHead: number) {
-        if (this.serviceConfiguration.scribe.nackMessages.enable &&
-            this.serviceConfiguration.scribe.nackMessages.maxOps >= (this.sequenceNumber - this.protocolHead)) {
-            // we were over the limit, so we must have been nacking messages
-            // tell deli to stop
-            await this.sendNackMessage(undefined);
+        if (this.serviceConfiguration.scribe.nackMessages.enable) {
+            const opsSincePreviousSummary = this.sequenceNumber - this.protocolHead;
+            if (opsSincePreviousSummary >= this.serviceConfiguration.scribe.nackMessages.maxOps) {
```

Comment:
Just verifying my understanding- this comparison was backwards before, right?

And since this is called in response to new summary, it would only think it was previously nacking after the 2nd summary (or several possibly, with low maxOps) because the time between this summary and last summary must have been smaller than maxOps to realize it needs to stop nacking?

Reply:
Exactly